interface-definitions/include/openconnect-config-per-x.xml.i

```diff
@@ -0,0 +1,54 @@
+<!-- include start from openconnect-config-per-x.xml.i -->
+<node name="config-per-x">
+    <properties>
+        <help>Configures ocserv to search the configured directory for a config file matching the Group name or username</help>
+    </properties>
+    <children>
+        <leafNode name="mode">
+            <properties>
+                <help>ocserv will ignore these configs if groupconfig is enabled TODO: explain this in better detail - need to make it clear this is in reference to the groupconfig vyos config that lives up one level from here</help>
```

Comment:
Before there's a final version of the wording, perhaps it should be a draft PR. ;)

Reply:
AH, so in my workflow I add a bunch of these TODO's and work through them one by one and when I commit changes I diff them and review them and search for 'TODO' to make sure I got them all.

I must have missed this one - I didn't intend to submit a PR with this in it, I'll get that fixed up.